Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM python:2.7
+FROM python:3.6
```

Comment:
I suggest to change to python:3.8 or at least 3.7.
Support for python 3.6 ended on Dec 2021.